src/pages/contact.js

```diff
@@ -50,6 +160,34 @@ class ContactUs extends Component {
     e.preventDefault()
     this.setState({ submitting: true })
 
+    const strippedState = Object.assign({}, this.state)
+    delete strippedState.name
+    delete strippedState.email
+    delete strippedState.message
+    delete strippedState.submitting
+    delete strippedState.triedSubmitting
+    delete strippedState.privacy
+
+    // if you tick and untick a field, it remains in state as 'false'; we want ot filter those out
+    const strippedFilteredState =
+      Object.keys(strippedState).length > 0
+        ? Object.keys(strippedState).filter(item => {
+            return strippedState[item]
+          })
+        : []
+
+    const branches =
+      strippedFilteredState.length > 0
+        ? strippedFilteredState.reduce((acc, current) => {
+            return (acc = [...acc, this.state[current].branch])
+          }, [])
+        : ['engineering']
+
+    const uniqueBranches = Array.from(new Set(branches))
+
+    const chosenBranch =
+      uniqueBranches.length > 1 ? 'engineering' : uniqueBranches[0]
```

Comment:
default this to engineering

Reply:
... it already defaults to engineering? If there are two unique branches, it will go to engineering, otherwise it will pick engineering or community, based on the unique branches[0] - a few lines above here branches is already set to engineering by default.